### DISCUSSION TITLE
Creating and managing chroots

### DISCUSSION THREAD

**Discussion by alkisg** [ltsp/ltsp#200] [Q&A]
The older LTSP 5 had a tool called `ltsp-build-client` that administrators could use to create chroots, and another tool called `ltsp-chroot` to manage them.

The new LTSP doesn't provide them anymore, as nowadays numerous alternatives to manage chroots exist.

I created a wiki page for administrators that want to build and maintain chroots, using `debootstrap` and `kvm`. It works even on headless servers, via SSH etc:

[https://github.com/ltsp/ltsp/wiki/chroots](https://github.com/ltsp/ltsp/wiki/chroots)

**Comment by dptech06** [comment]
Hello,

I see this URL is 404 error:  https://github.com/ltsp/community/wiki/chroots

How can I manage whit new version ?

Thank for your help,
Sincerely,
David

**Comment by alkisg** [comment]
The community project was merged with the ltsp project, see:
https://github.com/ltsp/ltsp/wiki/chroots

**Reply by dptech06** [reply]
Yes ! Perfect !
Thank a lot
David P.
